ccmlib/scylla_repository.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import with_statement
+
+import os
+import tarfile
+import tempfile
+import time
+import shutil
+import glob
+
+from six import print_
+from six.moves import urllib
+
+from ccmlib.common import (ArgumentError, CCMError, get_default_path, rmdirs, validate_install_dir)
+from ccmlib.repository import __download
+
+GIT_REPO = "http://github.com/scylladb/scylla.git"
+
+RELOCATABLE_URLS = {
+    'unstable/master': 'https://s3.amazonaws.com/downloads.scylladb.com/relocatable/unstable/master/'
```

Comment:
Do we need to have a map between the versions and their location?
I'm not sure I understand this part and the setup part...

Reply:
Yeah this mapping can be remove, I wasn't clear to me at the time where would be the locations of the packages, and I had only examples from unstable/master.

I can use a base url and try to append to that

I'm still missing where formal version would be located

Reply:
I've removed the mapping, and now it's a string template.

what's not clear about the setup part ? the version related code ? or something else ?

the idea that I wanted that we could have things like:
```bash
# version format
# [branch]:[job id | "latest"]
unstable/master:234  # specific job version, only thing we support now
3.1:latest # latest version available for branch
3.1:22 # specific job for branch
```

Reply:
Now it's more clear to me.
I didn't understand the version part.

Eventually we will need to have a map of official versions that we can pull automatically (like cassandra has) so we can test things like upgrades inside dtest.